pandas/_libs/tslibs/offsets.pyi

```diff
@@ -200,37 +205,54 @@ class FY5253Mixin(SingleConstructorOffset):
     ) -> None: ...
 
 class FY5253(FY5253Mixin): ...
-class FY5253Quarter(FY5253Mixin): ...
+
+class FY5253Quarter(FY5253Mixin):
+    def __init__(
+        self,
+        n: int = ...,
+        normalize: bool = ...,
+        weekday: int = ...,
+        startingMonth: int = ...,
+        qtr_with_extra_week: int = ...,
+        variation: str = ...,
+    ) -> None: ...
+
 class Easter(SingleConstructorOffset): ...
 
 class _CustomBusinessMonth(BusinessMixin):
     def __init__(
         self,
         n: int = ...,
         normalize: bool = ...,
-        offset: timedelta = ...,
+        weekmask: str = ...,
         holidays: None | list = ...,
-    ): ...
+        calendar: np.busdaycalendar = ...,
```

Comment:
Looking at the `__init__()` in the `pyx` file, I think `calendar` should be `None | np.busdaycalendar`